src/storage/StorageHttpHandler.cpp

```diff
@@ -139,5 +195,45 @@ std::string StorageHttpHandler::toStr(folly::dynamic& vals) const {
     return ss.str();
 }
 
+bool StorageHttpHandler::downloadSSTFiles(const std::string& url,
+                                          int port,
+                                          const std::string& path,
+                                          const std::vector<std::string>& parts,
+                                          const std::string& local) {
+    for (auto& part : parts) {
+        std::stringstream ss;
+        ss << "hdfs://" << url << ":" << port << path << "/";
+        auto len = part.size();
+        if (len == 1) {
+            ss << "part-0000" << part;
+        } else if (len == 2) {
+            ss << "part-000" << part;
+        } else if (len == 3) {
+            ss << "part-00" << part;
+        } else if (len == 4) {
+            ss << "part-0" << part;
+        } else {
+            ss << "part-" << part;
+        }
```

Comment:
Just a suggestion,` folly::stringPrintf("part-%0*d%s", ...) ` instead of` if-else-if, etc. block` is better?